Class-Code/Elementary-CA/jkohl/sketch.js

```diff
@@ -1,14 +1,52 @@
-//Your Name(s)
-//Project Name
-//Date
-
-/* A brief description of what the program is */
-
```

Comment:
Make sure to include:
- Name
- Project Name
- Date
- A brief description of the code